src/coinbase/validator.ts

```diff
@@ -0,0 +1,64 @@
+import { GetValidator200ResponseValidator, ValidatorListData } from "../client";
+import { Coinbase } from "./coinbase";
+
+/**
+ *
+ */
+export class Validator {
+  /**
+   * Returns the list of Validators.
+   *
+   * @param networkId - The network ID.
+   * @param assetId - The asset ID.
+   * @param status - The status to filter by.
+   * @returns The list of Validators.
+   */
+  public static async list(
+    networkId: string,
+    assetId: string,
+    status?: string,
+  ): Promise<ValidatorListData> {
+    let validators: ValidatorListData = {};
+
+    const response = await Coinbase.apiClients.validator!.listValidators(
+      networkId,
+      assetId,
+      status,
+    );
+
+    if (networkId.startsWith("ethereum") && assetId === "eth") {
+      validators = response.data.data;
```

Comment:
are we going to have different schema for other networks?

Reply:
yes - different validators can potentially have very very different schema. For the near future, we will only have ethereum validators.